esphome/components/sensirion_common/__init__.py

```diff
@@ -0,0 +1 @@
+CODEOWNERS = ["@martgras"]
```

Comment:
Supporting change for changing parents of the sensors.

```suggestion
import esphome.codegen as cg

from esphome.components import i2c


CODEOWNERS = ["@martgras"]

sensirion_common_ns = cg.esphome_ns.namespace("sensirion_common")

SensirionI2CDevice = sensirion_common_ns.class_("SensirionI2CDevice", i2c.I2CDevice)
```